src/overlays/actors/ovl_En_Ruppecrow/z_en_ruppecrow.c

```diff
@@ -10,12 +10,38 @@
 
 #define THIS ((EnRuppecrow*)thisx)
 
+enum {
+    ENRUPPECROW_EFFECT_NONE = 0,
+    ENRUPPECROW_EFFECT_ICE = 10,
+    ENRUPPECROW_EFFECT_LIGHT = 20,
+};
+
 void EnRuppecrow_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnRuppecrow_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnRuppecrow_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnRuppecrow_Draw(Actor* thisx, GlobalContext* globalCtx);
+void EnRuppecrow_HandleSong(EnRuppecrow*, GlobalContext*);
+s32 EnRuppecrow_UpdateCollision(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_UpdateRupees(EnRuppecrow*);
+void EnRuppecrow_UpdateDamage(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_HandleDeath(EnRuppecrow*);
+void EnRuppecrow_FallToDespawn(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_ShatterIce(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_UpdatePosition(EnRuppecrow*, GlobalContext*);
+s32 EnRuppecrow_CheckPlayedMatchingSong(GlobalContext*);
+void EnRuppecrow_HandleSongCutscene(EnRuppecrow*, GlobalContext*);
+s32 EnRuppecrow_ReachedPointClockwise(EnRuppecrow*, Path*, s32);
+s32 EnRuppecrow_ReachedPointCounterClockwise(EnRuppecrow*, Path*, s32);
+f32 EnRuppecrow_GetPointDirection(Path*, s32, PosRot*, Vec3s*);
+s32 EnRuppecrow_CanSpawnBlueRupees(GlobalContext*);
+void EnRuppecrow_SpawnRupee(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_FlyWhileDroppingRupees(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_UpdateSpeed(EnRuppecrow*, GlobalContext*);
+void EnRuppecrow_FlyToDespawn(EnRuppecrow*, GlobalContext*);
+
+extern AnimationHeader D_060000F0;
+extern FlexSkeletonHeader D_060010C0;
```

Comment:
With objects now in, you can include the appropriate object header and use the autogenned symbols. In this case you want to include `objects/object_crow/object_crow.h` and convert
`D_060000F0 -> object_crow_Anim_0000F0`
`D_060010C0 -> object_crow_Skel_0010C0`
You would also delete the `ovl_En_Ruppecrow` section in `undefined_syms.txt`